website/docs/create-your-composite.mdx

```diff
@@ -0,0 +1,111 @@
+import Tabs from  '@theme/Tabs'
+import TabItem from  '@theme/TabItem'
+
+# Create your composite
+The second step in getting started with ComposeDB on Ceramic is creating your composite to serve as your database schema. In this guide, we will create your first composite.
+
+
+>💡 Before proceeding you must have [set up your environment](./set-up-your-environment) in the previous step
+
+## Overview
+A composite is your database schema for ComposeDB, which includes a collection of data models. Once created, your composite instructs your node which models to index and also allows your client to perform queries and mutations on these models.
+
+
+## Data Model Catalog
+The Data Model Catalog contains all models created by other ComposeDB developers. By reusing models from the marketplace in your composite, you can instantly share and reuse data with other applications. This brings native app data composability to Web3 -- no more API integrations.
+
+To list all models in the marketplace, run the following command:
+
+```jsx
+composedb model:list --table
+```
+<!-- curated model:list command -->
+
+Here, the flag `--table` will display the output in an organized table view and provide more details about each model’s functionality.
+
+**Response:** Below see a small snippet of the the output table. Each model has the following properties:
+
+-  `Name` - model name
+
+-  `Unique ID` - unique identifier (stream ID) for the model
+
+-  `Description` - description of the model’s functionality
+
+![Data Model Table](https://s3.us-west-2.amazonaws.com/secure.notion-static.com/dcee19b2-54b7-46b5-bfe3-9ea0a9dd5b7a/Screenshot_2022-11-16_at_14.59.07.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Content-Sha256=UNSIGNED-PAYLOAD&X-Amz-Credential=AKIAT73L2G45EIPT3X45/20221206/us-west-2/s3/aws4_request&X-Amz-Date=20221206T180011Z&X-Amz-Expires=86400&X-Amz-Signature=cf6fbcf46fca88ef34a039a5c5292275f9c3816e2d49dc9a1fe793f6c7b43e72&X-Amz-SignedHeaders=host&response-content-disposition=filename=%22Screenshot%25202022-11-16%2520at%252014.59.07.png%22&x-id=GetObject)
```

Comment:
Better add these kinds of assets to the website directly than importing from external URLs I think.